cuda_core/cuda/core/experimental/_module.py

```diff
@@ -250,6 +349,13 @@ def arguments_info(self) -> list[ParamInfo]:
         _, param_info = self._get_arguments_info(param_info=True)
         return param_info
 
+    @property
+    def occupancy(self) -> KernelOccupancy:
+        """Get the read-only attributes of this kernel."""
```

Comment:
nit: we already have `.attributes` above, let's disambiguate it
```suggestion
        """Get the occupancy information of this kernel."""
```